test/unit/EventCalendarTest.js

```diff
@@ -24,14 +65,1306 @@ describe('EventCalendar', () => {
+        });
+
+        it('should not show the previous month arrow', () => {
+            expect($html.find('.clndr-previous-button').hasClass('inactive')).to.be.true;
+        });
+
+        it('should not show the next month arrow', () => {
+            expect($html.find('.clndr-next-button').hasClass('inactive')).to.be.true;
+        });
+    });
+
+
+    describe('passing events to the init() method', () => {
+        beforeEach(() => {
+            eventCalendar.init('2018-07-01', null, [{ date: '2018-07-04' }]);
+        });
+        
+        it('should add the appropriate styling to the date', () => {
+            expect($html.find('[data-day="2018-07-04"]').parent().hasClass('event')).to.be.true;
+        });
+
+        it('should add the appropriate aria-label to the date', () => {
+            expect($html.find('[data-day="2018-07-04"]').attr('aria-label')).to.contain('Selected');
+        });
+    });
+
+
+    describe('choosing a previously unselected date', () => {
+        beforeEach(() => {
+            eventCalendar.init('2018-07-01');
+        });
+        
+        it('should add the appropriate styling to the target date', () => {
+            $html.find('[data-day="2018-07-04"]').click();
+            expect($html.find('[data-day="2018-07-04"]').parent().hasClass('event-add')).to.be.true;
+        });
+
+        it('should add the appropriate aria label the target date', () => {
+            $html.find('[data-day="2018-07-04"]').click();
+            expect($html.find('[data-day="2018-07-04"]').attr('aria-label')).to.contain('Selected. Event will repeat');
+        });
+
+        it('the datesToAdd and datesToDel arrays should be empty', () => {
+            let dates = eventCalendar.getDates();
+            expect(dates.datesToAdd.length).to.equal(0);
+            expect(dates.datesToDel.length).to.equal(0);
+        });
+        
+        it('should add to target date the array of datesToAdd', () => {
+            $html.find('[data-day="2018-07-04"]').click();
+            
+            let dates = eventCalendar.getDates();
+            expect(dates.datesToAdd.length).to.equal(1);
+            expect(dates.datesToDel.length).to.equal(0);
+        });
+
+        it('should show the toAdd information in the review panel', () => {
+            $html.find('[data-day="2018-07-04"]').click();
+
+            expect($html.find('.js-dates-to-add').attr('style')).to.equal('display: inline;');
+        });
+
+        it('should update the toAdd information with the current number', () => {
+            $html.find('[data-day="2018-07-04"]').click();
+
+            expect($html.find('.js-dates-to-add').html()).to.equal('1 day will be added');
+        });
+
+        it('should not show the toDel information in the review panel', () => {
+            $html.find('[data-day="2018-07-04"]').click();
+
+            expect($html.find('.js-dates-to-del').attr('style')).to.equal('display: none;');
```

Comment:
I have held off of leaving any lint related comments, I think this repo should perhaps be used as a testbed for implementing `eslint`, we could fine tune the config using this repo (or just go flat out and pick a pre-existing one), then we can use that in the main Pulsar repo.

The one linty comment I'll leave, is that it'd be nice if all the `expect` blocks and a blank line above like these ones.